nservicebus/rabbitmq/configuration-api.md

```diff
@@ -107,4 +107,25 @@ If the routing topologies mentioned above isn't flexible enough then take full c
  1. Define the topology by creating a class implementing `IRoutingTopology`
  1. Register it with the transport calling `.UseRoutingTopology` as shown below
 
-snippet:rabbitmq-config-useroutingtopology
\ No newline at end of file
+snippet:rabbitmq-config-useroutingtopology
+
+## Transactions and delivery guarantees
+
+### Version 6 and above
+
+The RabbitMQ transport supports the following [Transport Transaction Modes](/nservicebus/messaging/transactions.md):
+
+* Receive Only
+* No transactions
+
+### Receive Only
+
+When running in Receive Only mode, the RabbitMQ transport consumes messages from the broker in manual acknowledgment mode. After a message is successfully processed, it is acknowledged via the AMQP [basic.ack](http://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.ack) method, which lets the broker know that the message can be removed from the queue. If a message is not successfully processed and needs to be retried, it is requeued via the AMQP [basic.reject](http://www.rabbitmq.com/amqp-0-9-1-quickref.html#basic.reject) method.
+
+If the connection to the broker is lost for any reason before a message can be acknowledged, the message will automatically be requeued by the broker, letting it be retried by the endpoint.
```

Comment:
Hmm, I wasn't really thinking of this as being dangerous for this mode, (or needing an explicit call out) since we already talk about that here: http://docs.particular.net/nservicebus/messaging/transactions#transactions-transport-transaction-receive-only

I was under the impression that these transport-specific sections were supposed to call out things that were unique to how the transport handled it.